update the content of SVG `<view>` element (#33710)
mdn/content: files/en-us/web/svg/element/view/index.md

{{SVGRef}}

> The `<view>` SVG element defines a particular view of the SVG document. It is used by referencing the element's id as the target fragment of a URL.

Reviewer: ```suggestion
The `<view>` SVG element defines a particular view of an SVG document. A specific view can be displayed by referencing the `<view>` element's [`id`](/en-US/docs/Web/HTML/Global_attributes/id) as the target fragment of a URL.
```
